content/internships/history/contents.lr

```diff
@@ -29,6 +35,6 @@ CC has participated in Google Summer of Code eight times before 2020. See below
 
 ## Outreachy
 
-CC participated in the Dec 2019 to Mar 2020 round of Outreachy.  
+CC participated in the May 2020 to December 2020 round of Outreachy.  
```

Comment:
We participated in both rounds, this is the history page so it should include both.

Reply:
Resolved in f00524f.